server/routes.go

```diff
@@ -36,13 +34,16 @@ func InitRoutes(r *gin.RouterGroup) {
 
 	write.POST("/election/create", actions.CreateElection)
 	write.PUT("/election/:id/edit", actions.EditElection)
```

Comment:
`PATCH` rather than `PUT` here? PUT updates the whole resource (replaces it with the request body), PATCH updates only provided fields:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PATCH
https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT